improver/cli/snow_fraction.py

```diff
@@ -35,7 +35,7 @@
 
 @cli.clizefy
 @cli.with_output
-def process(*cubes: cli.inputcube):
+def process(*cubes: cli.inputcube, model_id_attr: str = None):
```

Comment:
Is there a reason this is a different CLI?  Are they ever used independently?

I can see good reasons for having separate plugins but if the CLIs are always used in conjunction, it would make sense to have just one CLI to call.

Reply:
This is a many to many problem. Rain & Snow => snow-fraction.  Snow-fraction => Rain & Sleet & Snow. We will definitely have to call the phase-mask CLI three times (once for each phase), but there's no need to recalculate the snow-fraction three times. It isn't the cheapest calculation in the world because of the interpolation it does to fill in the NaNs.

Reply:
Ah OK.  That makes sense.